### PR TITLE
Move NuGet authoring project Microsoft.FxCopAnalyzers over from dotne…

### LINKED ISSUE #251: Move NuGet authoring project Microsoft.FxCopAnalyzers over from dotnet/roslyn
Need to find and move the project that builds Microsoft.FxCopAnalyzers nuget package from dotnet.roslyn. It pulls in System.Runtime.Analyzers, System.InteropServices.Analyzers and Microsoft.AnalyzerPowerPack.

### PR DESCRIPTION
**Pull request by mavasani**
…t/roslyn

Generated FxCopAnalyzers nuget package will have a dependency range (currently >= 1.0.0 && < 1.2.0) for dependent packages.

Testing: Verified prerelease and release packages built locally and adding a nuget reference to the FxCopAnalyzers nupkg installs the dependent packages from nuget.

Fixes #251

### CODE REVIEW

File: src/FxCop/NuGet/FxCopAnalyzers.nuspec [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.CodeAnalysis.FxCopAnalyzers</id>
+    <summary>Microsoft FxCop Analyzers</summary>
+    <description>
+      Microsoft FxCop rules implemented as analyzers using the .NET Compiler Platform ("Roslyn").
+    </description>
+    <dependencies>
```
Comment (mavasani) [inline src/FxCop/NuGet/FxCopAnalyzers.nuspec]: Is this list correct?

**Comment by mavasani** [comment]
@tmeschter @genlu @srivatsn @lgolding please review.

**Comment by mavasani** [comment]
Screenshot below:

fxcopanalyzers

File: build/Targets/Analyzers.Versions.targets [hunk 2]
```diff
@@ -9,28 +9,42 @@
...
+
+    <ToolsSemanticVersion>0.0.0</ToolsSemanticVersion>
+    <ToolsPreReleaseVersion>beta1</ToolsPreReleaseVersion>
+  </PropertyGroup>
+
+  <!-- Version info for FxCop analyzer packages -->
+  <PropertyGroup>
+    <!-- Master FxCop analyzer package, which is just a collection of individual analyzer packages below. -->
+    <FxCopAnalyzersSemanticVersion>1.0.2</FxCopAnalyzersSemanticVersion>
+    <FxCopAnalyzersPreReleaseVersion>beta1</FxCopAnalyzersPreReleaseVersion>
```
Comment (genlu) [inline build/Targets/Analyzers.Versions.targets]: @michaelcfanning Are we gonna keep using FxCop's name from now on?

Comment (mavasani) [inline build/Targets/Analyzers.Versions.targets]: Note that this package has already been published during VS 2015 RTM - we are just publishing a newer package now.

File: src/FxCop/NuGet/FxCopAnalyzers.nuspec (hunk 1 above)
Comment (tmeschter) [inline src/FxCop/NuGet/FxCopAnalyzers.nuspec]: Looks right to me.

**Comment by tmeschter** [comment]
LGTM

**Comment by mavasani** [comment]
Thanks Tom.

**Pull request merged**